management/src/main/java/org/ehcache/management/registry/DefaultCollectorService.java

```diff
@@ -113,6 +121,20 @@ public void cacheRemoved(String alias, Cache<?, ?> cache) {
         EhcacheNotification.CACHE_REMOVED.name()));
   }
 
+  private void cacheCleared(String alias) {
+    collector.onNotification(
+      new ContextualNotification(
+        configuration.getContext().with("cacheName", alias),
+        EhcacheNotification.CACHE_CLEARED.name()));
+  }
+
+  private void registerClearNotification(String alias, Cache<?, ?> cache) {
+    OperationStatistic<ClearOutcome> clear = StatsUtils.findOperationStatisticOnChildren(cache,
```

Comment:
I assume this is safe from a GC point of view? Anything un-natural with the context stuff makes me jumpy.